dotcom-rendering/src/amp/components/Elements.tsx

```diff
@@ -212,7 +212,7 @@ export const Elements = (
 				);
 			case 'model.dotcomrendering.pageElements.YoutubeBlockElement':
 				return (
-					<YoutubeBlockComponent
+					<YoutubeBlockComponentAMP
```

Comment:
Could we include a little bit of detail on why this component had to be renamed?

Reply:
Great question :) I alluded to this in the PR description (perhaps a little too vague!)

> rename the amp youtube block component to prevent webpack issues when determining correct importable file

IIRC, without the file re-naming webpack had issues determining which file to import (the amp one or the web one). The result was an error with the youtube importable on the client (something like 'youtubeblockcomponent.importable does not exist'). It was all very confusing until @oliverlloyd pointed out the necessity for this file name change :) 

I think the component name change probably isn't necessary (in fact, I don't _think_ it is because I just re-tested with the component name unchanged and all seemed ok). However, given the distinction needed between web and amp I quite like the component name updated too though, what do you think?